docs/content/installation/using-the-jwt-token-docker-secret.md

```diff
@@ -18,51 +18,155 @@ This document explains how to use the NGINX Plus Ingress Controller image from t
+
+
+### Manifest deployment
+
+[Installling with Manfiets](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/)
+
+Manifest deployment example:
+
+```yaml
+spec:
+  serviceAccountName: nginx-ingress
+  imagePullSecrets:
+  - name: regcred
+  automountServiceAccountToken: true
+  securityContext:
+    seccompProfile:
+      type: RuntimeDefault
+#        fsGroup: 101 #nginx
+  containers:
+  - image: private-registry.nginx.com/nginx-ic/nginx-plus-ingress:3.1.1
+    imagePullPolicy: IfNotPresent
+    name: nginx-plus-ingress
+```
+
+Notice `imagePullSecrets` and `containers.image` lines to represent our kubernetes secret as well as the registry and version of the NGINX Ingress controller we are going to deploy.
+
+### Helm install method
+
+If you are using `helm` to install, you can install using two methods. First is the `helm` sources method. [Helm sources install](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/#managing-the-chart-via-sources "helm source install")
+
+1. Clone the nginxinc/kubernetes-ingress repository
+2. Change directory into deployents/helm-chart of the recently cloned repo.
+3. Modify the `values.yaml` file.
+
+We want to set a few lines for NGINX Plus Ingress controller to be deployed.
+
+1. Change to `nginxplus`
+2. Specify NGINX Ingress controller image to use.
+3. Specify a `imagePullSecretName` to pull from the private registry.
+
+
+### Ensure nginxplus is set to `true`
+```yaml
+## Deploys the Ingress Controller for NGINX Plus.
+nginxplus: true
+```
+
+### Specify the image to be used for deployment
+
+```yaml
+image:
+  ## The image repository of the Ingress Controller.
+  repository: private-registry.nginx.com/nginx-ic/nginx-plus-ingress
+
+  ## The tag of the
+  tag: 3.1.1
+```
+
+### Modify `imagePullSecretName` and type in the name of the secret (regcred per the above) you created in step 1.
+
+```yaml
+  serviceAccount:
+    ## The annotations of the service account of the Ingress Controller pods.
+    annotations: {}
+
+    ## The name of the service account of the Ingress Controller pods. Used for RBAC.
+    ## Autogenerated if not set or set to "".
+    # name: nginx-ingress
+
+    ## The name of the secret containing docker registry credentials.
+    ## Secret must exist in the same namespace as the helm release.
+    imagePullSecretName: regcred
+```
+
+You can now install NGINX Ingress controller with the `values.yaml` file modified.
+
+```bash
+helm install nicdev01 -n nginx-ingress --create-namespace -f values.yaml .
+```
+
+The above command will install NGINX Ingress controller in the `nginx-ingress` namespace. If the namespace does not exist, `--create-namespace` will create it. Using `-f values.yaml` tells `helm` to use the `values.yaml` file that we modified earlier with the settings we want to apply for our NGINX Ingress controller deployment.
+
+### Using the `helm` charts method:
```

Comment:
Is this an alternate Helm install method that is different than what begins on line 70?
Line 70 represents itself as THE Helm install method.  So it is kind of confusing to the novice to have this insert itself in as something that is different without much explanation.

Maybe an explanation like:  "Helm also supports overriding or defining values using the command line.  If you have an existing values files and want to override or test this method before committing to values files changes, you can insert the necessary values using the following single command line option."